pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/KafkaLowLevelStreamConfig.java

```diff
@@ -53,12 +65,36 @@ public String getBootstrapHosts() {
     return _bootstrapHosts;
   }
 
+  public int getKafkaBufferSize() {
+    return _kafkaBufferSize;
+  }
+
+  public int getKafkaSocketTimeout() {
+    return _kafkaSocketTimeout;
+  }
+
+  private int getConfigWithDefault(Map<String, String> configMap, String key, int defaultValue) {
```

Comment:
I think that it's better to indicate `int` to the function name since this is specific to getting an int value. e.g. `getIntConfigWithDefault`

Reply:
updated in new diff